org.eclipse.xtext.xbase.lib/src/org/eclipse/xtext/xbase/lib/IterableExtensions.java

```diff
@@ -977,4 +977,23 @@ public static int size(Iterable<?> iterable) {
 	public static <T> T max(final Iterable<T> iterable, Comparator<? super T> comparator) {
 		return IteratorExtensions.max(iterable.iterator(), comparator);
 	}
+
+    /**
+     * Returns <tt>true</tt> if this Iterable contains the specified element.
+     * More formally, returns <tt>true</tt> if and only if this Iterable contains
+     * at least one element <tt>e</tt> such that
+     * <tt>(o==null&nbsp;?&nbsp;e==null&nbsp;:&nbsp;o.equals(e))</tt>.
+     *
+     * @param iterable 
+     * 			the elements to test
+     * @param o 
+     * 			element whose presence in the Iterable is to be tested
+     * @return <tt>true</tt> if this Iterable contains the specified element
+     */
+	public static <T> boolean contains(Iterable<T> iterable, Object o) {
```

Comment:
Signature could be simplified to `Iterable<?>`

Reply:
done